lib/libutee/tee_api_private.h

```diff
@@ -8,6 +8,8 @@
 #include <tee_api_types.h>
 #include <utee_types.h>
 
+/* From tee_api.c */
+extern TEE_TASessionHandle tee_api_system_session;
```

Comment:
I guess that should have double underscores (`__tee_api_system_session`) because it is a global symbol for internal use and we don't want any clash with user-defined stuff.

@jenswi-linaro there are a few instances of this rule not being enforced, such as the `utee_` syscall wrappers. Should they be renamed? (that would be taken separately of this PR of course).

Reply:
@jforissier, yes those could be renamed. I guess there's quite a bit of that in other corners of libutee too, one small step at a time is the way to go here I think.

Reply:
Renamed.